frontend/src/components/landing.tsx

```diff
@@ -1,17 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Header from './header';
 import Footer from './footer';
 import ReadMore from './readmore';
+import { GoogleLogin, GoogleLogout } from 'react-google-login';
```

Comment:
Whenever you install your dependencies, do not install them globally. Install them locally. Inside the frontend folder, run `npm i -S react-google-login` to add it to the project's dependencies.